src/components/v2/V2Create/V2WarningBanner.tsx

```diff
@@ -36,7 +36,7 @@ export default function V2WarningBanner() {
             fontWeight: 400,
             textDecoration: 'underline',
           }}
-          href="https://info.juicebox.money/build/project-design"
+          href="https://info.juicebox.money/build/project-desig://info.juicebox.money/dev/build/basics"
```

Comment:
@filipv-eth are these the correct links?

Reply:
Definitely not, sorry about this. Appreciate the correction!